adhesive/telegram_bot.py

```diff
@@ -220,25 +219,22 @@ def format_draft_message():
 			event.client.http,
 			meta,
 			token=event.client.config['signal']['stickers']['signalstickers_api_key'],
-			# TODO use configured log levels for this i guess
-			test_mode=event.client.config['signal']['stickers'].get('signalstickers_api_test_mode', False),
 		)
 
 		async with anyio.create_task_group() as tg:
 			await tg.spawn(partial(draft_msg.edit, buttons=None))
 			await tg.spawn(processing_message.delete)
 
-		if status_code not in range(200, 300):
+		if status_code != 200:
```

Comment:
Why should a code like 204 No Content, 201 Created, or 202 Accepted, be considered unsuccessful?

Reply:
The server specifically returns a 200, but you are right, I've added a check on 2xx range

Reply:
I do not see said change.

Reply:
Please put back the `x not in range()` version to minimize the size of this changeset.